frontend/src/features/projects/dispose/hooks/stepperContext.tsx

```diff
@@ -18,26 +11,17 @@ export const StepperContext = createContext({} as any);
 export const StepContextProvider = (props: { children?: any }) => {
   // Use State to keep the values. Initial values are obtained from StepperContextProvider’s props.
   const [currentStatus, setCurrentStatus] = useState<IStatus>();
+  const [disposeWorkflowStatuses, setDisposeWorkflowStatuses] = useState<IStatus[]>();
   // Make the context object (or array)
-  const stepContext = { currentStatus, setCurrentStatus };
+  const stepContext = { currentStatus, setCurrentStatus, disposeWorkflowStatuses };
```

Comment:
the initial dispose steps only ever need to load the workflow once, so might as well do that here during initial context setup instead of the hook (which was resulting in multiple api calls in some situations).